src/components/draw.js

```diff
@@ -1,8 +1,12 @@
-const draw = (howManyHeroes, maxValue = 730) => {
-  const arr = new Array(howManyHeroes).fill(1);
-  const result = arr.map((randomValue) => randomValue * Math
-    .floor(Math.random() * maxValue) + 1);
-  return result;
+const draw = (howManyHeroes, maxValue = 100) => {
+  const array = new Array(maxValue).fill(1);
+  const excludeCards = [9, 16, 19, 21, 22, 27, 33, 46, 47, 50, 51, 54, 55, 59, 62, 64, 65, 67, 74,
+    77, 85, 86, 89, 90, 91, 94];
+  const randomArray = array.map((randomValue) => randomValue * Math
+    .floor(Math.random() * (maxValue - 1)) + 1);
```

Comment:
you wrote:

    const array = new Array(maxValue).fill(1);

so all values in array are filled with 1

    const randomArray = array.map((randomValue) => ...

what you call randomValue here is an element of array: randomValue will always be 1, so it's not random...

try this:

    const invalidCards = [
      9, 16, 19, 21, 22, 27,
      33, 46, 47, 50, 51, 54,
      55, 59, 62, 64, 65, 67,
      74, 77, 85, 86, 89, 90,
      91, 94
    ];
    // list valid cards
    const validCards = [];
    for (let i = 1; i <= maxValue; i++) {
      if (!invalidCards.includes(i)) {
        validCards.push(i);
      }
    }
    // shuffle valid cards
    validCards.sort(() => Math.random() - 0.5);
    // slice first cards
    return validCards.slice(0, howManyHeroes);

Reply:
ok